app/scripts/modules/core/src/help/HelpField.tsx

```diff
@@ -41,6 +42,13 @@ export class HelpField extends React.PureComponent<IHelpFieldProps> {
     }
   };
 
+  private considerContext(fromContext: any, fromProps: any) {
```

Comment:
this method and its usage is confusing to me, probably because of the naming.

Might be easier to understand if it was something like 
```
shouldExpandHelpText(expandFromContext, expandFromProps) {}
```